crates/shared/src/price_estimation.rs

```diff
@@ -163,6 +163,13 @@ pub struct Arguments {
     /// trade and enables metrics accordingly.
     #[clap(long, env, action = clap::ArgAction::Set, default_value = "false")]
     pub enable_quote_predictions: bool,
+
+    /// Determines how confident we want to be that we quote the best price.
+    /// If the value is < 1 we may skip sending quote requests to price
+    /// estimators that have returned some winning quote in the past for a given
+    /// trade.
```

Comment:
I don't understand this sentence.

Reply:
I updated the `--help` text to explain what the feature is supposed to do in it's final form which is hopefully clearer. This exact behavior (`>= 1` meaning always asking every estimator) needs to be implemented in a follow up PR to ensure that we can completely disable this optimization in case it causes problems.